migrations/20190129083947-add-collective-country_iso.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const size = 32;
```

Comment:
Why `32`?

Reply:
Oh sorry, I just saw that I put `32` in the example I sent you, my bad. 

The correct value is `Sequelize.STRING(2)` as we only need to chars.